### ISSUE TITLE
Getting Started note is incorrect

### ISSUE THREAD

**Issue by cmcknight** [rothsandro/eleventy-notes#26]
The current instructions in the Getting Started note read:

```
Then clean up the existing notes by deleting all files and folders except for the .app folder.
```

If these instructions are followed exactly, the site will not work because the app.json file is required. Rather than assume the reader knows that, I would suggest that the text be modified to read:

```
Then clean up the existing notes by deleting all files and folders except for the .app folder and the app.json file.
```

**Comment by rothsandro** [comment]
> If these instructions are followed exactly, the site will not work because the app.json file is required.

While most users will probably create a new `app.json` later, it's not required. The app works without the configuration file as there are defaults defined for all values. Did you have problems starting the app without the file?

**Comment by cmcknight** [comment]
Yes, no note content showed up until I restored the app.json file. I would see the note pop up in the sidebar, but when I clicked on the note, nothing happened even though the URL in address bar was showing the note was selected.

Not sure exactly what was happening. 🤷‍♂️

**Comment by rothsandro** [comment]
Hm, strange. I can't reproduce this.

- Do you have a reproduction?
- Do you have any errors in the terminal?
- What browser, OS and npm/node version do you use?

**Comment by cmcknight** [comment]
Just did some more research (helps that I’ve had both sleep and coffee).

Javascript Console errors:

localhost/:1 Uncaught SyntaxError: Unexpected token 'l', "light" is not valid JSON
    at JSON.parse (<anonymous>)
    at (index):10:70
localhost/:1 Uncaught SyntaxError: Unexpected token 'l', "light" is not valid JSON
    at JSON.parse (<anonymous>)
    at storageGet (module.esm.js:33:15)
    at Alpine.interceptor.func.as.alias (module.esm.js:8:51)
    at Object.initialize (module.esm.js:348:14)
    at module.esm.js:332:26
    at Array.forEach (<anonymous>)
    at recurse (module.esm.js:327:59)
    at initInterceptors (module.esm.js:340:10)
    at Object.store (module.esm.js:1390:3)
    at aFXxm../alpine (appearance.js:3:8)
reload-client.js:21 [11ty][18:35:29.849 UTC] Connected

Node version: 20.2.0
Npm version: 9.6.6
MacOS: 13.4.1

After fiddling around with screen widths, it seems that the note content will not appear if the screen is less than 1200px. Not sure if that’s by design or by accident.

Steps to Reproduce

Clone repo (git clone https://github.com/rothsandro/eleventy-notes.git)
Remove all files and folders at the root level
Run npm “start” script (npm start)
Create new note at root of project and add text
Size screen to less than 1200px

Best regards,

Chuck

**Comment by rothsandro** [comment]
Thanks for the infos. It should work if you clean up your localStorage (on localhost).

Looks like you have a `theme` entry with the value `light` from another project that was running in localhost. Eleventy Notes also uses the `theme` entry but expects the value to be a JSON (e.g. `"light"` with quotes). Parsing the invalid value fails and the app crashes. On screens < 1200px this leads to the mobile navigation not hiding anymore and so you never see the note.

**Comment by rothsandro** [comment]
I created a new issue #27 to improve this behavior.

**Comment by cmcknight** [comment]
Thanks, that will doubtless help others who hit this issue and have no clue as to what is going on.

Best regards,

Chuck

**Comment by rothsandro** [comment]
Thanks for bringing this up. It's fixed in v0.12.1
